compiler-rt/test/memprof/TestCases/profile_reset.cpp

```diff
@@ -0,0 +1,39 @@
+// Test to ensure that multiple rounds of dumping, using the
+// __memprof_profile_reset interface to close the initial file
+// and cause the profile to be reopened, works as expected.
+
+// RUN: %clangxx_memprof  %s -o %t
+
+// RUN: rm -f %t.log.*
+// RUN: %env_memprof_opts=print_text=true:log_path=%t.log %run %t
+
+// Check both outputs, starting with the renamed initial dump, then remove it so
+// that the second glob matches a single file.
+// RUN: FileCheck %s --dump-input=always < %t.log.*.sv
```

Comment:
Is the dump-input left over from debugging? The default is on fail which seems fine.

Reply:
Looks like this came in when I copied from another test, where it is presumably leftover debugging. I removed from here.